07-sampling.Rmd

```diff
@@ -64,137 +60,119 @@ library(scales)
 
 ```{r eval=FALSE}
 ggplot(tactile_prop_red, aes(x = prop_red)) +
   geom_histogram(binwidth = 0.05, boundary = 0.4, color = "white") +
-  labs(x = "Proportion of 50 balls that were red", 
-       title = "Distribution of 33 proportions red") 
+  labs(x = "Proportion of red balls in each sample", 
+       title = "Histogram of 33 proportions") 
 ```
-```{r samplingdistribution-tactile, echo=FALSE, fig.cap="Distribution of 33 proportions based on 33 samples of size 50.", fig.height=3.1, purl=FALSE}
+```{r samplingdistribution-tactile, echo=FALSE, fig.cap="The distribution of sample proportions based on 33 random samples of size 50.", fig.height=3.1, purl=FALSE}
 tactile_histogram <- ggplot(tactile_prop_red, aes(x = prop_red)) +
   geom_histogram(binwidth = 0.05, boundary = 0.4, color = "white")
 tactile_histogram +
   labs(
-    x = "Proportion of 50 balls that were red",
-    title = "Distribution of 33 proportions red"
+    x = "Proportion of red balls in each sample",
+    title = "Histogram of 33 proportions of red balls for each sample"
   )
 ```
 
 
-### What did we just do? {#sampling-what-did-we-just-do}
-
-What we just demonstrated in this activity is the statistical concept of \index{sampling} *sampling*. We would like to know the proportion of the bowl's balls that are red. Because the bowl has a large number of balls, performing an exhaustive count of the red and white balls would be time-consuming. We thus extracted a *sample* of 50 balls using the shovel to make an *estimate*. Using this sample of 50 balls, we estimated the proportion of the *bowl's* balls that are red to be 34%.
+### Recap, some terminology, and additional remarks {#sampling-what-did-we-just-do}
 
-Moreover, because we mixed the balls before each use of the shovel, the samples were randomly drawn. Because each sample was drawn at random, the samples were different from each other. Because the samples were different from each other, we obtained the different proportions red observed in Figure \@ref(fig:samplingdistribution-tactile). This is known as the concept of *sampling variation*. \index{sampling!variation}
+The activity introduced the concept of \index{sampling} *sampling*. We wanted to know the proportion of red balls in a bowl. Since the bowl had many balls, performing an exhaustive count of red and white balls was impractical. Instead, we first took a *sample* of 50 balls and determine that the proportion of red balls in this sample was 34%. For simplicity, we say the *sample proportion* was 34%. This is an *estimate* of the proportion of red balls in the entire bowl. We then took more samples and obtain more sample proportions, 33 in total. This activity is what in statistics we call *sampling*. 
```

Comment:
```suggestion
The activity introduced the concept of \index{sampling} *sampling*. We wanted to know the proportion of red balls in a bowl. Since the bowl had many balls, performing an exhaustive count of red and white balls was impractical. Instead, we first took a *sample* of 50 balls to determine that the proportion of red balls in this sample was 34%. For simplicity, we say the *sample proportion* was 34%. This is an *estimate* of the proportion of red balls in the entire bowl. We then took more samples to obtain more sample proportions, 33 in total. This activity is what in statistics we call *sampling*. 
```